cmd/todd-agent/main.go

```diff
@@ -51,6 +51,11 @@ func main() {
 
 	cfg := config.GetConfig(arg_config)
 
+	// Create optDir
+	if err := os.MkdirAll(cfg.LocalResources.OptDir, 0755); err != nil {
```

Comment:
So, the first part of my question is - how do supporting directories like ```/opt/todd``` get created? I have yet to get a straight answer as to how we could still create such things in the absence of a Makefile. I'd love to hear your response to https://github.com/Mierdin/todd/issues/30. To your point, even if ```go install``` could do such things, it's not meant to be run as sudo, and the default location of the optdir requires this to be created. I feel like I'm in a war between what is go-idiomatic, and what is linux-idiomatic.

Until I get a good answer to that issue, I see no reason to use ```go install```, and would prefer to use ```make install```.